src/app/modules/shared_models/keypair_model.nim

```diff
@@ -75,10 +75,16 @@ QtObject:
         return self.items[i]
     return nil
 
-  proc onUpdatedAccount*(self: KeyPairModel, keyUid, address, name, colorId, emoji, prodPreferredChainIds, testPreferredChainIds: string) =
+  proc onUpdatedAccount*(self: KeyPairModel, keyUid, address, name, colorId, emoji: string) =
     for item in self.items:
       if keyUid == item.getKeyUid():
-        item.getAccountsModel().updateDetailsForAddressIfTheyAreSet(address, name, colorId, emoji, prodPreferredChainIds, testPreferredChainIds)
+        item.getAccountsModel().updateDetailsForAddressIfTheyAreSet(address, name, colorId, emoji)
+        break
```

Comment:
`return` could be better option instead of `break` (you don't need to change, just saying)

Reply:
I think both should do the same thing in this case, so i'll let it be.